.mise.toml

```
@@ -131,3 +135,10 @@ find . -type f -name "Tiltfile" | while read -r file; do
     black --check --diff "$file"
   done
 """
+
+[tasks."bootstrap:kubeconfig"]
+description = "Set KUBECONFIG"
+run = "mise set KUBECONFIG=$(pwd)/.mise/kubeconfig.yaml"
+
+[hooks]
+enter = [ "mise bootstrap:kubeconfig" ]
```

To reduce risk of accidentally committing unnecessary vars to `.mise.toml`, I think we should do this:
```suggestion
[tasks."bootstrap:kubeconfig"]
description = "Set KUBECONFIG"
run = "mise set --env local --file mise.local.toml KUBECONFIG=$(pwd)/.mise/kubeconfig.yaml"

[hooks]
enter = ["mise bootstrap:kubeconfig"]
```

And then also add `mise.local.toml` to the `.gitignore`